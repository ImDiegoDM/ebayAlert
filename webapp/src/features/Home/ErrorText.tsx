import styled from 'styled-components';
export const ErrorText = styled.div`
  margin-top: 10px;
  color: red;
`;
