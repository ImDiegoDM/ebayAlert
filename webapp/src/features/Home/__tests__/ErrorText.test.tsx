import { render } from '@testing-library/react';
import React from 'react';
import { ErrorText } from '../ErrorText';

describe('<ErrorText/>', () => {
  it('should match snapshot', () => {
    const { container } = render(<ErrorText/>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
