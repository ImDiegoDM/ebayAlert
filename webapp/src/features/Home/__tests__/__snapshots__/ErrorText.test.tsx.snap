// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorText/> should match snapshot 1`] = `
.c0 {
  margin-top: 10px;
  color: red;
}

<div
  class="c0"
/>
`;
